<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSciSearchBar</class>
 <widget class="QWidget" name="QSciSearchBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>77</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QPushButton" name="replaceBtn">
     <property name="text">
      <string>&amp;Replace</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>findEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QCheckBox" name="caseBox">
     <property name="text">
      <string>Match case</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="replaceLabel">
     <property name="text">
      <string>Replace:</string>
     </property>
     <property name="buddy">
      <cstring>replaceEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="nextBtn">
     <property name="text">
      <string>&amp;Next</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="findEdit"/>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="replaceAllBtn">
     <property name="text">
      <string>Replace &amp;All</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="prevBtn">
     <property name="text">
      <string>&amp;Previous</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="closeBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 5px</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../actions.qrc">
       <normaloff>:/actions/red-cross</normaloff>:/actions/red-cross</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="replaceEdit"/>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="expandBtn">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/icons/arrow-top</normaloff>
       <normalon>:/icons/arrow-bottom</normalon>:/icons/arrow-top</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QCheckBox" name="wholeBox">
     <property name="text">
      <string>Whole word</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../actions.qrc"/>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
